torchdata/datapipes/iter/load/fsspec.py

```diff
@@ -101,11 +101,11 @@ def __iter__(self) -> Iterator[str]:
                             yield abs_path
 
 
-@functional_datapipe("open_file_by_fsspec")
+@functional_datapipe("open_files_by_fsspec")
```

Comment:
I think this one is BC breaking from 0.3.0. Should we add `open_file_by_fsspec` to this class as well?

Call `IterDataPipe. register_datapipe_as_function("open_file_by_fsspec", FSSpecFileOpenerIterDataPipe)` at the end of this file.

Reply:
Yea, we can do that.